released/packages/coq-mi-cho-coq/coq-mi-cho-coq.0.1/descr

```diff
@@ -0,0 +1,4 @@
+Mi-Cho-Coq is a formalisation of the Michelson language using the Coq
```

Comment:
Redundant with the `description` field of the `opam` file (file to remove).

Reply:
Sorry, are you saying that the descr file should be removed?

Reply:
Yes

Reply:
I have removed the two descr files.